integration_tests/commands/async/hmset_test.go

```diff
@@ -0,0 +1,42 @@
+package async
+
+import (
+	"gotest.tools/v3/assert"
+	"testing"
+)
+
+func TestHMSET(t *testing.T) {
```

Comment:
Please add a test case where you are setting multiple fields in a single `HMSET` command, then verify that all of those fields contain the new value. To make this test even more robust, some of these fields could be pre-existing, while others can be new.

Reply:
done, sorry for missing this.